scientific_library/tvb/simulator/models/reduced_wong_wang_exc_io_inh_i.py

```diff
@@ -0,0 +1,357 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could this be merged/linked somehow with wong_wang_exc_inh.ReducedWongWangExcInh model class, or should it stay as a complete separate implementation ? They seem to have a lot of common code

Reply:
Sure. I have done so already in the tvb-mulstiscale-cosim branch. Not needed to repeat here since anyway tvb-mulstiscale-cosim should be merged at some point to this branch, after this branch is merged to the master.